src/platform/portduino/architecture.h

```diff
@@ -17,3 +17,5 @@
 #ifndef HAS_TELEMETRY
 #define HAS_TELEMETRY 1
 #endif
+#define HAS_WIRE 1
+#define HAS_SCREEN 1
```

Comment:
Shouln"t it read #ifndef HAS_WIRE ... #endif to be in line with all other definitions in architecture.h and configuration.h

Reply:
The last commit is my in-progress work, hacks included, trying to get a screen to light up.